src/InArrayValidator.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace ipl\Validator;
+
+use ipl\I18n\Translation;
+
+class InArrayValidator extends BaseValidator
+{
+    use Translation;
+
+    /** @var array The array */
+    protected $haystack;
+
+    /** @var bool Whether the types of the needle in the haystack should also match */
+    protected $strict = false;
+
+    /**
+     * Validates whether the value exists in given haystack
```

Comment:
The constructor does not validate. And you can use Markdown. Once we render API docs, it makes sense to have them good looking. PHPStorm also renders Markdown:

```
Create a new validator with the given options

**Supported options:**

* `haystack`: (`array`) The array
* `strict`: (`bool`) Whether ..., default `false`